app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -35,10 +33,17 @@ import org.mozilla.focus.tiles.CustomHomeTile
 import org.mozilla.focus.tiles.CustomTilesManager
 import org.mozilla.focus.tiles.HomeTile
 import org.mozilla.focus.utils.OnUrlEnteredListener
+import android.view.ContextMenu
+import android.view.MenuItem
+import android.view.KeyEvent
+import kotlinx.android.synthetic.main.home_tile.view.*
+import org.mozilla.focus.ext.toUri
 
 private const val COL_COUNT = 5
 private const val SETTINGS_ICON_IDLE_ALPHA = 0.4f
 private const val SETTINGS_ICON_ACTIVE_ALPHA = 1.0f
+private const val bundled_blacklist = "blacklist"
+private const val HOME_TILES_PREFS = "homeTilesPrefs"
```

Comment:
This pref value has already been added to HomeTileManager - do not duplicate it here.

Please take a look at the code in HomeTileManager, and integrate your changes there. To keep the code clean (such as HomeFragment) we don't want to do logic in here that should be controlled by something like HomeTileManager. It makes the overall logic of HomeFragment harder to read, because 1) there is more code so you have to know what is important/not, and 2) it makes HomeTileManager also confusing because it's no longer the place to look for all the Home Tile code, some of it's hanging out here.

Reply:
^ just forgot to remove it, woopsss!!